README.md

```diff
@@ -5,16 +5,11 @@ This repo holds downloadable resources for use with [Snowplow Accelerators](http
 ### Folder Structure Example
 ```
 Web Analytics Accelerator
-|----   Snowflake
-|       |    Streamlit
-|       |    Looker
-|----   BigQuery
-|       |    Streamlit
+|    Streamlit
+|    Looker
 
 E-Commerce Accelerator
-|----   Snowflake
-|       |   Looker
-|----   BigQuery
-|       |   Streamlit
+|   Looker
```

Comment:
We'll need to change the instructions on downloading these repos in the actual accelerators as well. We can probably handle it in a similar way to the package versions in the template.

Reply:
Yeah I'm working through getting it all standardised and having some example data if possible for each of them now, and then everything should actually require the same versions of packages and follow the same naming, so we'll have like 6 PRs to go live at the same time. 

Nice bonus of this is I'm pretty sure it adds bigquery for all the snowflake only ones, except we need the data upload for them still - if we go down the public dataset route that adds them completely!